docker/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3"
 services:
   bootstrap:
     image: quorumcontrol/tupelo:${TUPELO_VERSION}
-    build: "${TUPELO_BUILD_PATH}"
+    entrypoint: ["/usr/bin/tupelo"]
```

Comment:
This isn't in the image already?

Reply:
It's this: https://github.com/quorumcontrol/tupelo/blob/master/Dockerfile#L31

Reply:
@cap10morgan we good to merge? I think the lack of the StoragePath is breaking the wasm tests.